src/client-api-proxy.ts

```diff
@@ -75,6 +75,7 @@ async function getData(
         scheduledSurface(id: $scheduledSurfaceId) {
           items(date: $date) {
             corpusItem {
+              id
```

Comment:
We need the scheduled item id up 1 level up. That id lets the data team correlate it to a surface and when it was scheduled.

Reply:
Good point, I misread the Slack thread - will update shortly. Thanks @bassrock